.jsbeautifyrc

```diff
@@ -0,0 +1,11 @@
+{
+  "eol": "\n",
+  "indent_char": " ",
+  "indent_size": 2,
+  "indent_with_tabs": false,
+  "brace_style": "none,preserve-inline",
+  "max_preserve_newlines": 2,
+  "wrap_line_length": 100,
```

Comment:
This project is using prettier 😄  and have different line width defaults (80) so I think adding this file could cause conflicts for people who have both formatters installed in their IDE 😨

Reply:
oops, sorry for this one, forgot to remove the file